lib/prosopite.rb

```diff
@@ -192,16 +192,16 @@ def mysql_fingerprint(query)
       query
     end
 
-    def send_notifications
-      @custom_logger ||= false
-      @rails_logger ||= false
-      @stderr_logger ||= false
-      @prosopite_logger ||= false
-      @raise ||= false
+    def clean_notifications(notifications)
+      notifications.map do |queries, kaller|
+        kaller = backtrace_cleaner.clean(kaller)
```

Comment:
backtrace_cleaner is also called in the `generate_notification_message` - should be removed from there